src/Components/IssueCard.jsx

```diff
@@ -277,7 +277,6 @@ const CardActions = ({
 
 const useStyles = makeStyles((theme) => ({
   container: {
-    'width': '29em',
```

Comment:
@pablo-mayrgundter I am not sure how to set the issue card size based on the outer container.
The issue cards are located in the notes panel, if the width of the notes panel is set explicitly it just scrolls in the y direction, because the body of the issue takes more space than the width of the notes panel.